process/tsc_faq.md

```diff
@@ -52,9 +61,9 @@ Roles we've seen in a project include but aren't limited to...
 
 ## Meetings
 
-TSC meetings are opportunities for the project community to come together to share updates and discuss issues and future plans. All TSC meetings are considered open to the public unless there is a sensitive issue to discuss ( in the case of private meetings, Linux Foundation staff should be present at such meetings ).
+TSC meetings are opportunities for the project community to share updates and discuss issues and plans. All TSC meetings are considered open to the public unless there is a sensitive issue to discuss ( in the case of private meetings, Linux Foundation staff should be present at such meetings ).
```

Comment:
The part about a LF representative needing to be present at non-public meetings might need a bit of clarification. What would be the process for requesting that? Has it happened before in ASWF?

Reply:
Not at ASWF, but it has happened in the cases of conduct or legal questions. I would say we would know about the meeting well beforehand ;-)